### PR TITLE
More updates to IModelDB and IObservableValue for strictNullCheck.

### PR DESCRIPTION
**Pull request by ian-r-rose**
Some additional updates to `coreutils` for use with `strictNullCheck`.

### CODE REVIEW

File: packages/cells/src/model.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ import {
 } from '@jupyterlab/coreutils';
 
 import {
-  IObservableJSON, IModelDB, IObservableValue, ObservableValue
```
Comment (blink1073) [inline packages/cells/src/model.ts]: I've been purposely leaving the notebook stack alone due to the refactoring.

Comment (ian-r-rose) [inline packages/cells/src/model.ts]: This was just renaming the `ObservableValue` namespace to `IObservableValue` for consistency with the other `IObservables`. I can revert that.

Comment (blink1073) [inline packages/cells/src/model.ts]: I'd say yes please.

Comment (ian-r-rose) [inline packages/cells/src/model.ts]: Reverted

**Pull request merged**